src/plugins/intel_cpu/src/nodes/scaled_attn.cpp

```diff
@@ -1071,6 +1071,16 @@ void ScaledDotProductAttention::createPrimitive() {
         if (rtPrecision == ov::element::bf16) {
 #ifdef OPENVINO_ARCH_X86_64
             executor = std::make_shared<AttentionExecutor<KT_ONEDNN, ov::bfloat16>>(context);
+#endif
+        } else if (rtPrecision == ov::element::f16) {
```

Comment:
Seems like it it makes inaccessible branch on line 1088.
Maybe this is the reason why Arm test is failing

Reply:
> Seems like it it makes inaccessible branch on line 1088. Maybe this is the reason why Arm test is failing

reorg this code snippet. 
new code looks more clear for different platforms.

Reply:
Thanks!